tests/functional/cli/test_cli.py

```diff
@@ -34,12 +34,6 @@ def test_config_error_with_help_prints_help(script_runner):
     assert ret.returncode == 0
 
 
-def test_global_help_prints_resources_vertically(script_runner):
-    ret = script_runner.run(["gitlab", "--help"])
-    assert """resource:\n  application\n  application-appearance\n""" in ret.stdout
```

Comment:
Should we check here that the endpoint is shown, maybe with a regex if a hard coded string is too brittle?

Reply:
My plan in the long term is to get rid of the other test and the VerticalHelp printer. As if I add `help=None` to arguments it does the same thing.

This is one step in that direction.

Reply:
Ah. I didn't read that close enough. Check for the endpoints in the test?

We could. Not sure if we need to.

Reply:
I just remember we've had some CLI regressions in the past, but if it's going away then it shouldn't matter :+1: